_episodes/05-merging-data.md

```diff
@@ -85,18 +85,18 @@ survey_sub = surveys_df.head(10)
 # Grab the last 10 rows
 survey_sub_last10 = surveys_df.tail(10)
 # Reset the index values to the second dataframe appends properly
-survey_sub_last10=survey_sub_last10.reset_index(drop=True)
+survey_sub_last10 = survey_sub_last10.reset_index(drop=True)
 # drop=True option avoids adding new index column with old index values
 ~~~
 {: .language-python}
 
 When we concatenate DataFrames, we need to specify the axis. `axis=0` tells
-pandas to stack the second DataFrame under the first one. It will automatically
+pandas to stack the second DataFrame UNDER the first one. It will automatically
```

Comment:
I guess you're suggesting to emphasize this fact. what if we bold-and-or-italicize it? uppercase seems screamy

Reply:
Actually, let's leave UPPERCASE and see what lesson learners say.

Reply:
It's used with other "directions" like `RIGHT` or `BEYOND` in 03...